includes/Story_Renderer/HTML.php

```diff
@@ -267,36 +267,22 @@ protected function print_analytics( $content ) {
 	}
 
 	/**
-	 * Print amp-story-bookend before closing `</amp-story>`.
+	 * Print amp-story-social-share before closing `</amp-story>`.
 	 *
-	 * @since 1.3.0
+	 * @since 1.6.0
 	 *
 	 * @param string $content String to replace.
 	 *
 	 * @return string
 	 */
-	protected function print_bookend( $content ) {
-		$share_providers = [
-			[
-				'provider' => 'twitter',
-			],
-			[
-				'provider' => 'linkedin',
-			],
-			[
-				'provider' => 'email',
-			],
-			[
-				'provider' => 'system',
-			],
-		];
+	protected function print_social_share( $content ) {
+		$share_providers = [ 'twitter', 'linkedin', 'email', 'system', ];
```

Comment:
Sadly the format of this array / object is different in the new component. Changing this format means it is not compatible with format of the old one and the filter. I think maybe we need to change the filter name here. Thoughts @swissspidy 

https://github.com/ampproject/amphtml/blob/d94427b5a9bc7ca1ef77718b2c4e25ed0469092e/extensions/amp-story/1.0/test/validator-amp-story-social-share.out#L34-L43